scripts/templates/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -36,7 +36,7 @@ metadata:
               "env": [
                 {
                   "name": "JAVA_OPTS",
-                  "value": "-Djava.util.prefs.userRoot=$(SONATYPE_WORK)/javaprefs"
+                  "value": "-Djava.util.prefs.userRoot=$(SONATYPE_WORK)/javaprefs -Djavax.net.ssl.trustStore=/etc/nexus-iq-server/jks/keystore.jks"
```

Comment:
always try to use the keystore.jks file we build. if it doesn't exist, the server will still start fine.